Source/Creation/Frame/Content/ID3FrameCreatorsChainFactory.swift

```diff
@@ -73,21 +161,58 @@ class ID3FrameCreatorsChainFactory {
             frameCreator: frameFromStringISO88591ContentCreator,
             id3FrameConfiguration: frameConfiguration
         )
+        let discPositionFrameCreator = ID3DiscPositionFrameCreator(
+            frameCreator: frameFromStringISO88591ContentCreator,
+            id3FrameConfiguration: frameConfiguration
+        )
+        let movementIndexFrameCreator = ID3MovementIndexFrameCreator(
+            frameCreator: frameFromStringISO88591ContentCreator,
+            id3FrameConfiguration: frameConfiguration
+        )
+        let seriesIndexFrameCreator = ID3SeriesIndexFrameCreator(
+            frameCreator: frameFromStringISO88591ContentCreator,
+            id3FrameConfiguration: frameConfiguration
+        )
         let recordingDateTimeCreator = ID3RecordingDateTimeFrameCreator(
             frameCreator: frameFromStringISO88591ContentCreator,
             id3FrameConfiguration: frameConfiguration,
             timestampCreator: ID3TimestampCreator()
         )
         albumFrameCreator.nextCreator = albumArtistCreator
         albumArtistCreator.nextCreator = artistFrameCreator
-        artistFrameCreator.nextCreator = titleFrameCreator
-        titleFrameCreator.nextCreator = yearFrameCreator
+        artistFrameCreator.nextCreator = commentFrameCreator
+        commentFrameCreator.nextCreator = composerFrameCreator
+        composerFrameCreator.nextCreator = conductorFrameCreator
+        conductorFrameCreator.nextCreator = contentGroupingFrameCreator
+        contentGroupingFrameCreator.nextCreator = copyrightFrameCreator
+        copyrightFrameCreator.nextCreator = encodedByFrameCreator
+        encodedByFrameCreator.nextCreator = encoderSettingsFrameCreator
+        encoderSettingsFrameCreator.nextCreator = fileOwnerFrameCreator
+        fileOwnerFrameCreator.nextCreator = fileTypeFrameCreator
+        fileTypeFrameCreator.nextCreator = groupingFrameCreator
+        groupingFrameCreator.nextCreator = languageFrameCreator
+        languageFrameCreator.nextCreator = lyricistFrameCreator
+        lyricistFrameCreator.nextCreator = mediaTypeFrameCreator
+        mediaTypeFrameCreator.nextCreator = mixArtistFrameCreator
+        mixArtistFrameCreator.nextCreator = movementNameFrameCreator
+        movementNameFrameCreator.nextCreator = movementIndexFrameCreator
+        movementIndexFrameCreator.nextCreator = podcastCategoryFrameCreator
+        podcastCategoryFrameCreator.nextCreator = podcastDescriptionFrameCreator
+        podcastDescriptionFrameCreator.nextCreator = podcastIDFrameCreator
+        podcastIDFrameCreator.nextCreator = podcastKeywordsFrameCreator
+        podcastKeywordsFrameCreator.nextCreator = publisherFrameCreator
+        publisherFrameCreator.nextCreator = subtitleFrameCreator
+        subtitleFrameCreator.nextCreator = titleFrameCreator
+        titleFrameCreator.nextCreator = unsyncedLyricsFrameCreator
+        unsyncedLyricsFrameCreator.nextCreator = yearFrameCreator
```

Comment:
Move your creator after mine in the chain (because they are less used and common in a lot of mp3 files).